src/neo-vm/ExecutionEngine.cs

```diff
@@ -278,6 +278,7 @@ private bool ExecuteInstruction()
                         break;
                     }
                 case OpCode.ABORT:
+                case OpCode.THROW:
```

Comment:
They are different. I think this commit 310ef93 should be reverted.

Reply:
What was the difference?

Reply:
`THROW` is catchable, `ABORT` is uncatchable.